oemoflex/facades.py

```diff
@@ -1,10 +1,16 @@
-from oemof.solph import sequence, Bus, Source, Sink, Transformer, Flow, Investment
+from oemof.solph import (
+    sequence, Bus,
+    Source as solph_Source,
+    Sink,
+    Transformer as solph_Transformer,
+    Flow, Investment
+    )
```

Comment:
Here, you could also just `from oemof import solph` and then use `solph.Transformer`. This would apply to the other objects as well, but that would perhaps lead to more clearity as well. What do you think?